src/window/settings.rs

```diff
@@ -32,6 +42,9 @@ pub struct Settings {
 
     /// The icon of the window.
     pub icon: Option<Icon>,
+
+    /// Platform specific settings.
+    pub platform_specific: platform::PlatformSpecific,
```

Comment:
Nit: redundant import path. `PlatformSpecific` is already in scope in this module.

Reply:
If i remove that line, what do you then want me to export here?

```rust
// src/window.rs
pub use settings::{PlatformSpecific, Settings};
```


```bash
error[E0432]: unresolved import `settings::PlatformSpecific`
 --> src/window.rs:9:20
  |
9 | pub use settings::{PlatformSpecific, Settings};
  |                    ^^^^^^^^^^^^^^^^ no `PlatformSpecific` in `window::settings`
  |
help: consider importing one of these items instead
  |
9 | pub use settings::{crate::Event::PlatformSpecific;
  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
9 | pub use settings::{crate::event::PlatformSpecific;
  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
9 | pub use settings::{crate::runtime::settings::PlatformSpecific;
  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
9 | pub use settings::{iced_native::Event::PlatformSpecific;
  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    and 4 other candidates
```